docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,28 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
```

Comment:
can you also write this information in a different place that will stay at 1.12? because if someone starts using CSI from 1.12, he won't know that those features exist in CSI

Reply:
what's new is always summarized also in the changelog.
there is also additional information within the user information itself.